components/buttonSelectors/NumberButtonSelectorQuickSurvey.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
```

Comment:
could you explain how this component is different from the NumberButtonSelector? I'm seeing a lot of repetition between the two files, so I'm wondering if this is an effective use of componentization.